packages/0x.js/src/types.ts

```diff
@@ -283,4 +285,8 @@ export interface OrderStateInvalid {
 export type OrderState = OrderStateValid | OrderStateInvalid;
 
 export type OnOrderStateChangeCallback = (err: Error | null, orderState?: OrderState) => void;
+
+export type OnTransactionSuccessCallback = (transactionReceipt?: TransactionReceiptWithDecodedLogs) => void;
+
+export type OnTransactionFailedCallback = (transactionReceipt?: TransactionReceipt) => void;
```

Comment:
And this one too

Reply:
Typical mistake copy and pasting, noted for fix.